docs/arch/scheduler.md

```diff
@@ -0,0 +1,60 @@
+# Scheduler
+
+## Contributors
+
+- DE MARTINO Giada Flora
+- DURAT Mathias
+- PLANCHE Benoit
+- TCHILINGUIRIAN Théo
+
+## Lexicon
+
+- Action: a CI atomic unit. It contains infrastructure, environment and commands to execute.
+- Agent: an agent is a computing node registered with the scheduler.
+- Agent pool: the set of all registered agents. It is a scheduler's entire knowledge about available computing resources.
+- Scheduling: selection of an agent on which to execute an action.
+- Action status: the state of the execution of an action (running, successful, failed).
+
+## The "What", "Why" and "How" of the scheduler
+
+### What?
+
+The scheduler receives a stream of CI actions (CI job units) as its main input. It also tracks a set of CI agents (registered computing nodes) that provide a dynamic resource pool, where the CI actions will be executed.
+The scheduler receives a stream of CI actions (CI job units) as its main input. It also tracks a set of CI agents (registered computing nodes) that provide a dynamic resource pool, where the CI actions will be executed.
```

Comment:
That is a duplicate of the previous sentence.

Reply:
:hurtrealbad: oops. Fixed :suspect: